sphinx/source/docs/user_guide/annotations.rst

```diff
@@ -153,15 +152,15 @@ information and examples.
 .. note::
     :ref:`userguide_interaction_legends` features currently work on the first,
     "per-glyph" style legends. Legends that are created by specifying a column
-    to automatically group do no yet support interactive features.
+    to automatically group do not yet support interactive features.
 
 .. _userguide_plotting_color_bars:
 
 Color Bars
 ----------
 
 A |ColorBar| can be created using a |ColorMapper| instance, which
-contains a color palette. Both on- and off-plot color bars are
+contains a color palette. Both on-plot and off-plot color bars are
 supported; the desired location can be specified when adding the
```

Comment:
As long as we are editing this we can get rid of the semicolon as well. Suggest more like:
> Colorbars may be added inside or outside of plots. This desires location is specified...